README.md

```diff
@@ -1,18 +1,50 @@
+- Use the [Appeals Status](https://developer.va.gov/explore/benefits/docs/appeals) API to request the status of a Veteran's benefits claim appeal. 
+
+### Benefits Claims
+- Use the [Claims](https://developer.va.gov/explore/benefits/docs/claims) (Auto-Establishment) API to submit a Veteran's benefits claim via EVSS (Electronic Veterans Self-Service), and to request the status of a Veteran's benefits claim.
+
+### Benefits Intake (fka Document Upload Service)
+- The [Benefits Intake](https://developer.va.gov/explore/benefits/docs/benefits) API allows authorized third-party systems used by Veteran Service Organizations and agencies to upload claim documents (scanned to PDF) directly to the Veterans Benefits Administration's (VBA) claims intake process.
+
+### Facilities
+- Use the [Facilities](https://developer.va.gov/explore/facilities/docs/facilities) API to find contact info, location, hours of operation, available services and other relevant information about a specific VA facility. 
+
+### Health (Dev only)
+- Use the [Health APIs](https://developer.va.gov/explore/health/docs/argonaut) to allow Veterans to view their medical records, schedule an appointment, find a specialty facility, and share their information with caregivers and providers.
+
+### Veteran Verification
+
+- The [Veteran Verification](https://developer.va.gov/explore/verification) API empowers Veterans to access their service history, Veteran status, discharge information and disability rating and put that information to work for them.
+
+#### Disability Rating
+- The [Disability Rating](https://developer.va.gov/explore/verification/docs/disability_rating) endpoint allows third-parties to access the disability rating of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+#### Service History
+- The [Service History](https://developer.va.gov/explore/verification/docs/service_history) endpoint allows third-parties to access the service history of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+#### Veteran Confirmation
+
+- The [Veteran Confirmation](https://developer.va.gov/explore/verification/docs/veteran_confirmation) endpoint allows third-parties to request confirmation from the VA of an individual's Veteran status after receiving authorization to do so using an Open ID Connect flow.
 
-### Document Upload Service
-- The [VBA document upload service](services/vba_documents) provides a 
-mechanism to submit VBA forms and documents to VA.
 
 ## Requesting Access
-We're still working to make these APIs generally available. In the meantime,
-please email `Alex.Yale-Loehr@va.gov` to request developer access. 
+If you have any questions or need further assistance, please email our team at api@va.gov. You will receive a human reply within 24 hours.  Additionally, you may open a comment or support request [inside this repo](https://github.com/department-of-veterans-affairs/vets-api-clients/issues/new/choose).
```

Comment:
This line is changing from an out-dated method of requesting access to a current method of obtaining help _other_ than requesting access. This should probably either include a link to the developer portal or have the header re-worded.

Reply:
agreed.  the header should be changed, as "requesting access" is all done via developer.va.gov/apply.